raiden-dapp/.stylelintrc

```diff
@@ -8,6 +8,7 @@
      {
        "ignorePseudoElements": ["v-deep"]
      }
-   ]
+   ],
+   "scss/selector-nest-combinators": "always"
  }
```

Comment:
Might make sense to apply a rule that checks if correct BEM naming is used as class names

Reply:
Done